client/src/components/Draftail/sources/ModalWorkflowSource.js

```diff
@@ -252,21 +259,26 @@ class LinkModalWorkflowSource extends ModalWorkflowSource {
 
       if (data.id) {
         if (data.parentId !== null) {
-          url = `${global.chooserUrls.pageChooser}${data.parentId}/`;
+          url = chooserUrl?.pageChooser
+            ? `${chooserUrl.pageChooser}${data.parentId}/`
+            : `${global.chooserUrls.pageChooser}${data.parentId}/`;
         } else {
-          url = global.chooserUrls.pageChooser;
+          url = chooserUrl?.pageChooser || global.chooserUrls.pageChooser;
         }
       } else if (data.url.startsWith('mailto:')) {
-        url = global.chooserUrls.emailLinkChooser;
+        url =
+          chooserUrl?.emailLinkChooser || global.chooserUrls.emailLinkChooser;
         urlParams.link_url = data.url.replace('mailto:', '');
       } else if (data.url.startsWith('tel:')) {
-        url = global.chooserUrls.phoneLinkChooser;
+        url =
+          chooserUrl?.phoneLinkChooser || global.chooserUrls.phoneLinkChooser;
         urlParams.link_url = data.url.replace('tel:', '');
       } else if (data.url.startsWith('#')) {
-        url = global.chooserUrls.anchorLinkChooser;
+        url =
+          chooserUrl?.anchorLinkChooser || global.chooserUrls.anchorLinkChooser;
         urlParams.link_url = data.url.replace('#', '');
       } else {
-        url = global.chooserUrls.externalLinkChooser;
+        url = chooserUrl.externalLinkChooser;
```

Comment:
here I just forget to add `global.chooserUrls` ,
it must be `url =  chooserUrl?.externalLinkChooser || global.chooserUrls.externalLinkChooser; `